packages/core/src/dataTypes/dateonly/coerce.spec.ts

```diff
@@ -3,60 +3,60 @@ import { HatchifyCoerceError } from "../../types/index.js"
 
 describe("coerce", () => {
   it("handles undefined", () => {
-    expect(() => coerce(undefined, { type: "Date" })).toThrow(
+    expect(() => coerce(undefined, { type: "Dateonly" })).toThrow(
       new HatchifyCoerceError("as a non-undefined value"),
     )
   })
 
   it("handles null", () => {
-    expect(coerce(null, { type: "Date", allowNull: true })).toBeNull()
-    expect(() => coerce(null, { type: "Date", allowNull: false })).toThrow(
+    expect(coerce(null, { type: "Dateonly", allowNull: true })).toBeNull()
+    expect(() => coerce(null, { type: "Dateonly", allowNull: false })).toThrow(
       new HatchifyCoerceError("as a non-null value"),
     )
   })
 
   it("handles non-strings", () => {
-    expect(() => coerce(1, { type: "Date" })).toThrow(
-      new HatchifyCoerceError("as an ISO 8601 date string"),
```

Comment:
i changed this error message because an iso 8601 date string can contain timezone, but we _only_ want yyyy-mm-dd